ntripclient.c

```diff
@@ -566,6 +569,32 @@ static int encode(char *buf, int size, const char *user, const char *pwd)
   return bytes;
 }
 
+static ssize_t curl_send_(CURL *curl, const void *buf, size_t len, int flags)
+{
+    (void)flags; // not used
+    size_t bytes_sent = 0;
+    CURLcode result = curl_easy_send(curl, buf, len, &bytes_sent);
+    return (result == CURLE_OK) ? bytes_sent : -1;
+}
+
+static ssize_t curl_recv_(CURL *curl, void *buf, size_t len, int flags)
+{
+    (void)flags; // not used
+    size_t bytes_recv = 0;
+    CURLcode result = CURLE_OK;
+
+    do
+    {
+        result = curl_easy_recv(curl, buf, len, &bytes_recv);
+
+        // Wait for 100ms before trying again.
+        if (result == CURLE_AGAIN)
+            usleep(100 * 1000);
```

Comment:
consider defining 1ms to remove magic number:
```
unsigned int one_ms = 1000;
...
usleep(100 * one_ms);
```

Reply:
Done. Made a macro for SLEEP_MS.